aws/cloudformation/standalone/marketing/deployment/marketing.yml.erb

```diff
@@ -503,14 +505,17 @@ Resources:
   # End AWS EFS Resources   #
   ###########################
 
-  ###########################
-  # Start AWS ECS Resources #
-  ###########################
+  ##########################################
+  # Start Github Action Deployer Resources #
+  ##########################################
 
   MarketingGithubDeployerUser:
     Type: AWS::IAM::User
     Properties:
-      UserName: marketing-github-deployer-development-cfn
+      UserName: !Sub
+        - "github-deployer-${SubdomainName}.${RootDomainName}"
+        - RootDomainName:
+            Fn::ImportValue: !Sub "${MarketingRootDnsStackName}-RootHostedZoneAddress"
```

Comment:
We should create the Deployer IAM user outside of this "application" stack. That user should be created by an admin, I want this stack to be able to be Created (not just Updated) by this user.